### ISSUE TITLE
Split GetHostEntry(IPAddress) into its own sample

### ISSUE THREAD

**Issue by GuardRex** [dotnet/docs#2583]
Updates were made to accommodate the `GetHostEntry(IPAddress)` overload in the sample on https://github.com/dotnet/docs/pull/2550 so that the API ref would show a proper (method call) example for this overload and not the example for `GetHostEntry(String)` (hostname as `String`), which was JustWrong:tm:.

Updates were made to:
*samples/snippets/cpp/VS_Snippets_Remoting/System.Net.Dns/CPP/dnsnewmethods.cpp*
*samples/snippets/csharp/VS_Snippets_Remoting/System.Net.Dns/CS/dnsnewmethods.cs*
*samples/snippets/visualbasic/VS_Snippets_Remoting/System.Net.Dns/vb/dnsnewmethods.vb*

However, this overload is best shown via its own sample as a complete executable example in the API coverage. Create a new set of samples to demo this overload and tie the new samples into *xml/System.Net/Dns.xml*.

**Comment by mairaw** [comment]
@GuardRex did we finish this one?

**Comment by GuardRex** [comment]
@mairaw No, @rpetrusha said to assign it to him if I didn't have time to get it done by 7/31. I was going to wait until next week to see where we're at before pulling the trigger on that.

**Comment by GuardRex** [comment]
@mairaw U saw ur E-mail, right? I list all of these in an update message that I sent last night.

**Comment by GuardRex** [comment]
@rpetrusha I won't get to this one by Monday night, so I'm handing over the reigns to you now.
